### PR TITLE
Fixes for 0.8.x

### PR DESCRIPTION
**Pull request by vtt-lair**
The roll table results are returned in a slightly different structure from 0.8.x. Changed the way this is handled for 0.8.x releases.

### CODE REVIEW

**Comment by akrigline** [comment]
Thanks for this! I haven't looked at LMRTFY in 0.8 yet, do you know if anything else is broken?

**Comment by vtt-lair** [comment]
Custom Formula is broken, it just shows the formula instead of the roll and result. I had a chat with Atropos on Discord on a possible solution which I'm testing. It's working but the speaker stays the player's name instead of the actor's name. Have tried playing around with a few things but no luck on that as yet (at least have it to a point where it's rolling again)

And then some midi-qol compatibility changes that Tim also mentioned in #94

**Comment by vtt-lair** [comment]
ANd just as I say that, I get it working!

@akrigline, do you mind if I had that to the same pull request? Everything I fix for 0.8.x instead of just the Roll Tables? Then I'll implement #94 changes as well.

**Comment by akrigline** [comment]
Yeah that sounds great to me, many thanks @vtt-lair

**Comment by vtt-lair** [comment]
Ok, this pull request now includes:

1. Changes to roll table result. Basically the data is returned one level lower now.
2. Changes to custom formula rolls. Refactored the code to use the new actor#getRollData function and use the roll#toMessage with create: false to generate the roll. Making sure the speaker is correct and send it to the ChatMessage.
3. Implement Tim's requested change on #94 

From my testing, that seems to be everything that needs to change for 0.8.x. Everything else seems to be working.

**Comment by Varriount** [comment]
Hm, this looks ok, however I'm unsure whether it's safe to merge this in.

File: src/roller.js
Comment (Varriount) [inline src/roller.js:199]: This appears to be trailing whitespace that should be cleaned up.

**Comment by akrigline** [comment]
Since installations happen from release artifacts, merging this to test further should be safe. I've recruited @vtt-lair as a maintainer too fyi @Varriount

**Comment by vtt-lair** [comment]
I've fixed the trailing empty space that @Varriount pointed out. Will go ahead and merge this in if neither of you disgree @akrigline , @Varriount

**Review by Varriount — APPROVED** [review]

**Pull request merged**
